File: Python PowerPoint Examples/PropertiesApiDemo/getPptDocumentProperty.py
import spirecloudpowerpoint
from spirecloudpowerpoint.configuration import *

appId = "your id"
appKey = "your key"
baseUrl = "https://api.e-iceblue.cn"
configuration = Configuration(appId, appKey, baseUrl)
api = spirecloudpowerpoint.api.properties_api.PropertiesApi(configuration)

name = "Property.pptx"
propertyName = "Author" #Author/Company/Format/LastSavedTime/Title
password = ""
storage = ""
folder = "input"

result = api.get_ppt_document_property(name, propertyName, password=password, storage=storage,
                                            folder=folder)

File: PHP PowerPoint Examples/DocumentPropertiesApiDemo/getPptDocumentProperty.php
<?php

use Spire\Cloud\Powerpoint\Sdk\Api\PropertiesApi;
use Spire\Cloud\Powerpoint\Sdk\Configuration;

$appId = "your id";
$appKey = "your key";
$baseUrl="https://api.e-iceblue.cn";
$configuration = new Configuration($appId, $appKey,$baseUrl);
$apiInstance = new PropertiesApi($configuration);

$name = "Property.pptx";
$password = null;
$folder = "input";
$storage = null;
$propertyName = "Author";
$result = $apiInstance->getPptDocumentProperty($name,$propertyName, $password, $folder, $storage);
print_r($result);